crates/distribution-types/src/installed.rs

```diff
@@ -114,6 +114,7 @@ impl InstalledDist {
     pub fn metadata(&self) -> Result<pypi_types::Metadata23> {
         let path = self.path().join("METADATA");
         let contents = fs::read(&path)?;
+        // TODO(zanieb): Update this to use thiserror so we can unpack parse errors downstream
         pypi_types::Metadata23::parse_metadata(&contents)
             .with_context(|| format!("Failed to parse METADATA file at: {}", path.user_display()))
```

Comment:
How likely is this to fail?

I'd rather not push more into this pull request but we panic on this later because we don't have the metadata error.